.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: ruby
+cache:
+  directories:
+    - vendor/bundle
+    - .gradle
+rvm:
+  - jruby-1.7.24
+script: rake bootstrap test:install-default && rake test:core test:plugins
```

Comment:
@andrewvc the idea is to start with a _clean_ environment, we could also achieve this by cleaning the hole workspace. Happy to see ways to achieve this with you.